symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/test/java/com/symphony/bdk/bot/sdk/monitoring/SymphonyHealthIndicatorTest.java

```diff
@@ -0,0 +1,79 @@
+package com.symphony.bdk.bot.sdk.monitoring;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.symphony.bdk.bot.sdk.lib.restclient.RestClient;
+import com.symphony.bdk.bot.sdk.lib.restclient.RestClientConnectionException;
+import com.symphony.bdk.bot.sdk.lib.restclient.RestClientImpl;
+import com.symphony.bdk.bot.sdk.symphony.HealthcheckClientImpl;
+
+import clients.SymBotClient;
+import configuration.SymConfig;
+import configuration.SymConfigLoader;
+import io.github.resilience4j.bulkhead.BulkheadConfig;
+import io.github.resilience4j.circuitbreaker.CircuitBreakerConfig;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.web.client.RestTemplate;
+
+import java.time.Duration;
+import java.util.Map;
+
+public class SymphonyHealthIndicatorTest {
+
+  private SymphonyHealthIndicator symphonyHealthIndicator;
+  private HealthcheckClientImpl healthcheckClient;
+  private SymBotClient symBotClient;
+  private RestClient restClient;
+
+  @Before
+  public void initSymphonyHealthIndicator(){
+    this.initRestClient();
+    this.initSymBotClient();
+    this.healthcheckClient = new HealthcheckClientImpl(this.symBotClient, this.restClient);
+    this.symphonyHealthIndicator = new SymphonyHealthIndicator(this.healthcheckClient);
+  }
+
+  @Test
+  public void testHealth(){
+    final Health health = this.symphonyHealthIndicator.health();
```

Comment:
Could you as well test the value of `health.getStatus()`? There are two possible values (UP and DOWN), so please create another test case for the second value.

Reply:
When the method `checkOverallStatus()` returns `false` we can verify the status is `Down`, but I can't find the necessary condition for status to be `UP`.

Reply:
Simplest way would be to mock to `HealthcheckClient` and pass the mock instance to the constructor of  `SymphonyHealthIndicator`. This way, you won't have to instantiate `SymBotClient` and `RestClient`.

Reply:
I don't understand the link with my problem ...

Reply:
That's good, sorry, I've just understood ;)